tests/sklearn/test_sklearn_models.py

```diff
@@ -81,20 +75,11 @@
 # sufficiently number of bits for precision
 N_BITS_THRESHOLD_FOR_SKLEARN_CORRECTNESS_TESTS = 26
 
-# We check correctness with check_is_good_execution_for_cml_vs_circuit or predict in
-# fhe="disable" only if n_bits >= N_BITS_THRESHOLD_FOR_PREDICT_CORRECTNESS_TESTS. This is
-# because we need sufficiently number of bits for precision
-N_BITS_THRESHOLD_FOR_PREDICT_CORRECTNESS_TESTS = 6
```

Comment:
removes this as I believe it was not very relevant : it was only used for test that were checking CML clear vs CML FHE/simulation, but N_BITS should not impact such behavior. Also, that won't change anything in regular CIs as they use at least 6 bits anyway. On the other hand, it enables KNNClassifier to be tested on things that were skipped previously because of this (since KNNClassifier can only be tested for n_bits <= 5 when compiled) cc @kcelia